src/Umbraco.Core/Mapping/MapperContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+
+namespace Umbraco.Core.Mapping
+{
+    /// <summary>
+    /// Represents a mapper context.
+    /// </summary>
+    public class MapperContext
```

Comment:
well we have to stop at one point. we cannot invent names for everything just because it may collide with automapper (else we'd have the same issue with our DI `IFactory` etc) - having our own `Mapper` class is annoying soon as you start using AutoMapper - OTOH this context thing will only be involved during our own mapping operations, so I don't think it's an issue